maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/IntegrationTestMojo.java

```diff
@@ -283,6 +288,22 @@
     @Parameter( property = "failsafe.runOrder", defaultValue = "filesystem" )
     private String runOrder;
 
+    /**
+     * When using a random order with <code>runOrder</code> parameter it is useful to be able to pass a seed.
+     * <br/>
+     * <br/>
+     * This is because, when tests fails to execution order binding, one can use the seed printed in
+     * reports to execute tests with exact same order as they where when they failed. This effectively reproduce
+     * the error. To reproduce a given seed most useful is to use command line option:
+     * <br/>
+     * <br/>
+     * <code>-Dfailsafe.randomSeed=325119</code>
+     *
+     * @since 2.19.2
+     */
+    @Parameter( property = "failsafe.randomSeed", defaultValue = Randomizer.DEFAULT_SEED )
+    private String randomSeed;
```

Comment:
Can we include this in the already existing parameter `runOrder`?
For instance, the usage can be `-Dfailsafe.runOrder=random:325119`.